apps/main/src/dex/components/ComboBoxSelectToken/index.tsx

```diff
@@ -133,23 +129,7 @@ const ComboBoxTokens = ({
 const StyledSpinnerWrapper = styled(SpinnerWrapper)`
   height: 100%;
   border: 0.5px solid var(--input_button--border-color);
-  box-shadow: inset -2px -2px 0px 0.25px var(--box--primary--shadow-color);
+  box-shadow: inset -2px -2px 0 0.25px var(--box--primary--shadow-color);
```

Comment:
how did this sneak in lol, i dont this changes anything functionally? 🤣 
dont mind either way

Reply:
WebStorm :fearful: